drivers/led/ltc3208/README.rst

```diff
@@ -0,0 +1,80 @@
+LTC3208 no-OS driver
+====================
+
+.. no-os-doxygen::
+
+Supported Devices
+-----------------
+
+`LTC3208 <https://www.analog.com/en/products/ltc3208.html>`_
+
+Overview
+--------
+
+The LTC3208 is a highly integrated multidisplay LED
+controller. The part contains a 1A high efficiency, low noise
+charge pump to provide power to the MAIN, SUB, RGB, CAM
+and AUX LED displays. The LTC3208 requires only small
+ceramic capacitors and one current set resistor to form a
+complete LED power supply and current controller
+
+The maximum display currents are set by a single external resistor. 
+Current for each LED is controlled by a precision internal 
+current source. Dimming and On/Off for all displays is achieved 
+via the I2C serial interface. 
+256 brightness levels are available for the MAIN and SUB displays. 
+16 levels are available for the RGB and CAM displays. 
+Four AUX current sources can be independently
+assigned via the I2C port to the CAM, SUB, MAIN or AUX
+DAC controlled displays.
+
+Applications
+------------
+
+* Video/Camera Phones with QVGA + Displays
+
+LTC3208 Device Configuration
+----------------------------
+
+Driver Initialization
+---------------------
+
+In order to be able to use the device, you will have to provide the support for
+the communication protocol (I2C) as mentioned above.
+
+The first API to be called is **LTC3208_init**. Make sure that it returns 0,
```

Comment:
please rework this file, function names are wrong, they should not use LTC (caps) prefix